stack/base-with-services/before-notebook.d/20_start-postgresql.sh

```diff
@@ -17,19 +15,15 @@ if [ ! -d /home/${NB_USER}/.postgresql ]; then
    echo "unix_socket_directories = '/tmp'" >> /home/${NB_USER}/.postgresql/postgresql.conf
    ${MAMBA_RUN} ${PSQL_START_CMD}
 
-# else don't
 else
     # Fix problem with kubernetes cluster that adds rws permissions to the group
     # for more details see: https://github.com/materialscloud-org/aiidalab-z2jh-eosc/issues/5
     chmod g-rwxs /home/${NB_USER}/.postgresql -R
 
-    # stores return value in $?
-    running=true
-    ${MAMBA_RUN} ${PSQL_STATUS_CMD} || running=false
-
-    # Postgresql was probably not shutdown properly. Cleaning up the mess...
-    if ! $running ; then
-       echo "" > /home/${NB_USER}/.postgresql/logfile # empty log files
+    if ! ${MAMBA_RUN} ${PSQL_STATUS_CMD}; then
+       # Cleaning up the mess if Postgresql was not shutdown properly.
+       # TODO: Rotate logfile
```

Comment:
Hi @danielhollas, what is this means, do you gonna to address this before it get merged?

Reply:
I've actually addressed this in #457, have a look.

Reply:
Make sense.